### DISCUSSION TITLE
Wio Link | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1082] [Q&A]
# Wio Link | Seeed Studio Wiki

What's the toughest part of building IoT applications? Someone says the jump wires usually make him frustrated, while the other guy says he hates soldering the most. There are even some people who don't like breadboard. Maybe you are not one of them, but the knowledge of electronic engineering, micro-controller programming, network programming, IoT protocols handling are still great burden between you and a successful IoT project.

https://wiki.seeedstudio.com/Wio_Link/

**Comment by thirumsk** [comment]
android app not available in the play store. IoS App "connection error". Wio link app is working or not?
